nilearn/maskers/base_masker.py

```diff
@@ -763,3 +770,67 @@ def set_output(self, *, transform=None):
             This has not been implemented yet.
         """
         raise NotImplementedError()
+
+
+def generate_lut(labels_img, background_label, lut=None, labels=None):
```

Comment:
extracted from niftilabelmasker to be used in both niftilabelmasker and surfacelabelmasker